app/code/Magento/Backend/Block/Widget/Grid/Column.php

```diff
@@ -337,6 +337,9 @@ public function getRowFieldExport(\Magento\Framework\DataObject $row)
             $renderedValue = call_user_func($frameCallback, $renderedValue, $row, $this, true);
         }
 
+        $renderedValue = str_replace("\r\n", " > ", trim($renderedValue));
```

Comment:
@bhatti-dhara I don't think this is the right approach to "fix" this issue. Actually, I don't see this PR addressing any valid issue.